src/lib/analytics/utils.ts

```diff
@@ -1,10 +1,13 @@
 import geoip from 'geoip-lite';
+import DeviceDetector from 'node-device-detector';
 const logger = require('../../lib/logger').default;
 const Logger = logger.getInstance();
 import { Location, User } from './types';
 // PROVISIONAL until express server typescript
 import express from 'express';
 
+const deviceDetector = new DeviceDetector;
```

Comment:
this works?

Reply:
Docs: https://www.npmjs.com/package/node-device-detector

```const DeviceDetector = require('node-device-detector');
const detector = new DeviceDetector;
const userAgent = 'Mozilla/5.0 (Linux; Android 5.0; NX505J Build/KVT49L) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/43.0.2357.78 Mobile Safari/537.36';
const result = detector.detect(userAgent);
console.log('result parse', result);